opsdir-review-template.md

```diff
@@ -1,16 +1,13 @@
 # ops-dir Review Template
 
-Hello, 
+Hi, 
 
 I have been selected as the Operations Directorate (opsdir) reviewer for this draft. 
 
-The Operations Directorate reviews all operational and management-related drafts to ensure alignment with operational best practices.
+The Operations Directorate reviews Internet-Drafts to ensure alignment with operational best practices and that adequate operational considerations are covered.
```

Comment:
Agree that opsdir reviews all I-Ds. But do we need to add further clarification to "ensure alignment with operational best practices"? Don't operational best practices already call for due operational considerations?

Reply:
That part is to help focus/remind the scope: this is about OPS matters.